docs/README.md

```diff
@@ -0,0 +1,27 @@
+# Gatsby Starter MDX Basic
```

Comment:
Should probably update this with information about contributing to the docs at some point.